_examples/onewire-relay/application.c

```diff
@@ -0,0 +1,49 @@
+#include <application.h>
+#include <bcl.h>
+#include <bc_module_sensor.h>
```

Comment:
And these two should be be needed whatsoever since they should be included in bcl.h :-)